Update guide Creating a regular order from an existing cart (#2186)
vtexdocs/dev-portal-content: docs/guides/Checkout/creating-a-regular-order-from-an-existing-cart.mdx

Before proceeding any further, ensure that you have a valid `appKey` and `appToken` granted with the necessary permissions to access the [Checkout](https://developers.vtex.com/docs/api-reference/checkout-api) and [Payments Gateway](https://developers.vtex.com/docs/api-reference/payments-gateway-api) APIs. For more information, please refer to the [Roles](https://help.vtex.com/en/tutorial/roles--7HKK5Uau2H6wxE1rH5oRbc) article.

## Preparing the cart

> In addition to valid credentials, make sure that the cart you will use to create the order already contains all the required information. Without these details, the steps described in this guide cannot be completed:

Reviewer: ```suggestion
In addition to valid credentials, ensure that the cart used to create the order already includes all the required information. Without these details, the steps described in this guide cannot be completed:
```
